scss/elements/badges.scss

```diff
@@ -0,0 +1,75 @@
+@mixin span-style($color, $background, $option, $width: 50%) {
+  position: absolute;
+  top: 0;
+  width: $width;
+  color: $color;
+  background-color: $background;
+
+  @if $option == left {
+    left: 0;
+  } @else if $option == right {
+    right: 0;
+  }
+}
+
+@mixin badge-style($color, $background, $is-splited: true) {
+  $box-shadow-first-two: 0 0.5em $background, 0 -0.5em $background;
+
+  @if $is-splited == true {
+    &:first-child {
+      @include span-style($color, $background, left);
+
+      box-shadow: $box-shadow-first-two, 0 0 $background, -0.5em 0 $background;
+    }
+    &:last-child {
+      @include span-style($color, $background, right);
+
+      box-shadow: $box-shadow-first-two, 0.5em 0 $background, 0 0 $background;
+    }
+  } @else {
+    &:first-child {
+      @include span-style($color, $background, 0, 100%);
+
+      box-shadow: $box-shadow-first-two, 0.5em 0 $background, -0.5em 0 $background;
+    }
+  }
+}
+
+// Default style
+.nes-badge {
+  $em: 0.75em;
+
+  position: relative;
+  display: inline-block;
+  width: $em * 14;
+  height: $em;
+  padding: 0.75em;
+  margin: 0.5em;
+  font-size: $em * 1.2;
+  text-align: center;
+  white-space: nowrap;
+  vertical-align: middle;
+  cursor: $cursor-click-url, pointer;
```

Comment:
`corsur` is unnecessary because `generic.scss` has the following definition. 🙏 
https://github.com/nostalgic-css/NES.css/blob/98ff32ad1e14accab0da049774c35482d4ac0325/scss/base/generic.scss#L27-L30

In the case of `span` etc, you cannot click `nes-badge`.
so you do not need to turn it into `pointer` 🙇

Reply:
Done.